src/pages/Account/Account.styles.tsx

```diff
@@ -0,0 +1,264 @@
+import { ExternalLink } from 'react-feather'
+import { Flex, Link, Text } from 'rebass'
+import styled from 'styled-components'
+
+import { ButtonPurpleDim } from '../../components/Button'
+import Logo from '../../components/Logo'
+import { ListLayout } from '../../ui/ListLayout'
+
+export const Status = styled(Flex)<{ status: string }>`
+  text-transform: uppercase;
+  padding: 4px;
+  font-size: 9px;
+  border-radius: 6px;
+  font-weight: 700;
+  border-width: 1.5px;
+  ${({ status }) => {
+    switch (status) {
+      case 'COMPLETED':
+      case 'CONFIRMED':
+      case 'CLAIMED':
```

Comment:
In my opinion it looks better. What do you think ?
```suggestion
       case "COMPLETED" || "CONFIRMED" || "CLAIMED":
```